tests/cephadm/test_cephadm_custom_config_support.py

```diff
@@ -0,0 +1,55 @@
+from cli.cephadm.cephadm import CephAdm
+from cli.exceptions import CephadmOpsExecutionError
+from cli.ops.cephadm import generate_bootstrap_config
+from cli.utilities.containers import Container
+
+
+def check_custom_file(node, config):
+    """Check custom file updated to conatiner"""
+    for spec in config["spec"]:
+        service = spec["service_name"]
+        container_id = (
```

Comment:
Use `format` parameter, it print ctr id. Refer - https://docs.podman.io/en/stable/markdown/podman-ps.1.html#format-format

Reply:
Thanks @vamahaja 
Updated